File: MyBreak/DayThree/DayFour/JavaScript/Chapter4TaskFive.js
function tabularOutputs(){
  let firstNumber = 0;
  let secondNumber = 0;
  let thirdNumber = 0;

  console.log("N  \tN2  \tN3  \tN4")
  for(let index = 0; index < tabs.length; index){
    firstNumber = index * index 
    secondNumber = index * index * index
    thirdNumber = index * index * index * index
    console.log(index + " \t" + firstNumber + " \t" + secondNumber + " \t" + thirdNumber)
  }
}

File: MyBreak/DayThree/DayFour/Python/chapter4_task_five.py

def tabular_output():
  first_number = 0
  second_number = 0
  third_number = 0

  print("N  \tN2  \tN3  \tN4")

  for index in range(6):
    first_number = index * index 
    second_number = index * index * index
    third_number = index * index * index * index

    print(index + "\t" + second_number + "\t" + third_number)
tabular_output